prov/efa/src/rdm/efa_rdm_ope.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2023 Amazon.com, Inc. or its affiliates.
```

Comment:
Can you expand this commit message... I feel like you do a lot more in this commit than:
`prov/efa: rename rxr_pkt_rx_map to efa_rdm_rxe_map`

Reply:
Sure. Will add more comments

Reply:
Updated comments